src/main/scripts/src/app/core/header/header.component.html

```diff
@@ -11,18 +11,21 @@
 
 <div class="menu">
   <div class="project" *ngIf="workspaceService.isWorkspaceSet()">
-    <a [routerLink]="['/workspace']">
+    <a tabindex [routerLink]="['/workspace']">
```

Comment:
Are these changes really needed? I think the TAB button works well even without them.

Reply:
`<a>` elements without `tabindex` or `href` in new Bootstrap change color to that of regular text on hover.. so instead of messing with Bootstrap styles I rather added it.